scrapy/utils/reactor.py

```diff
@@ -54,7 +54,7 @@ def __call__(self):
         return self._func(*self._a, **self._kw)
 
 
-def get_asyncio_event_loop_policy():
+def set_asyncio_event_loop_policy():
```

Comment:
For backward compatibility reasons, we should keep the old function around, and log a deprecation warning when called.